app/assets/stylesheets/base/_buttons.scss

```diff
@@ -1,33 +1,40 @@
-button, .button,
-input[type="submit"] {
-  @extend %button;
-  @include appearance(none);
-  border: none;
-  cursor: pointer;
-  user-select: none;
-  vertical-align: middle;
-  white-space: nowrap;
-
-  &.small {
-    padding: 0.5em .75em;
-    background-color: $small-button-color;
-
-    &:hover {
-      background-color: $small-hover-button-color;
-    }
-  }
-
-  &.success {
-    background-color: $success-button-color;
-    &:hover {
-      background-color: $success-hover-button-color;
-    }
-  }
-
-  &.warning {
-    background-color: $warning-button-color;
-    &:hover {
-      background-color: $warning-hover-button-color;
-    }
-  }
-}
+button, .button,
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own line